src/github.com/matrix-org/dendrite/clientapi/threepid/invites.go

```diff
@@ -25,14 +25,17 @@ import (
 	"time"
 
 	"github.com/matrix-org/dendrite/clientapi/auth/authtypes"
-	"github.com/matrix-org/dendrite/clientapi/auth/storage/accounts"
 	"github.com/matrix-org/dendrite/clientapi/producers"
 	"github.com/matrix-org/dendrite/common"
 	"github.com/matrix-org/dendrite/common/config"
 	"github.com/matrix-org/dendrite/roomserver/api"
 	"github.com/matrix-org/gomatrixserverlib"
 )
 
+type invitesAccountsData interface {
```

Comment:
I would change each instance of `*AccountsData` and `*DevicesData` to `*AccountData` and `*DeviceData`, just so the variable names make more grammatical sense.